{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/common.py

```diff
@@ -219,7 +219,7 @@ class Common(Configuration):
     GRAPPELLI_ADMIN_TITLE = '{{ cookiecutter.project_name }} Admin'
 
     EMAIL_SUBJECT_PREFIX = '[{{ cookiecutter.project_name }}]'
-    DEFAULT_FROM_EMAIL = '{{ cookiecutter.email }}'
+    DEFAULT_FROM_EMAIL = values.Value('{{ cookiecutter.email }}')
```

Comment:
Please change to [EmailValue](https://django-configurations.readthedocs.org/en/stable/values/#configurations.values.EmailValue)